.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
```

Comment:
I read that you should add a blank line before lists, but apparently, there is no fixed consensus about this. It's a minor thing.